demos/python_demos/whiteboard_inpainting_demo/whiteboard_inpainting_demo.py

```diff
@@ -0,0 +1,184 @@
+    return mask
+
+
+def remove_background(img, kernel_size=(7, 7), blur_kernel_size=21, invert_colors=True):
+    bgr_planes = cv2.split(img)
+    result = []
+    kernel = np.ones(kernel_size, np.uint8)
+    for plane in bgr_planes:
+        dilated_img = cv2.morphologyEx(plane, cv2.MORPH_OPEN, kernel)
+        dilated_img = cv2.dilate(dilated_img, kernel)
+        bg_img = cv2.medianBlur(dilated_img, blur_kernel_size)
+        if invert_colors:
+            diff_img = 255 - cv2.absdiff(plane, bg_img)
+        else:
+            diff_img = cv2.absdiff(plane, bg_img)
+        result.append(diff_img)
+    return cv2.merge(result)
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Whiteboard inpainting demo')
+    parser.add_argument('-i', type=str, help='Input sources (index of camera \
+                        or path to a video file)', required=True)
+    parser.add_argument('-mi', '--m_instance_segmentation', type=str, required=False,
+                        help='Path to the instance segmentation model')
+    parser.add_argument('-ms', '--m_semantic_segmentation', type=str, required=False,
+                        help='Path to the semantic segmentation model')
+    parser.add_argument('-t', '--threshold', type=float, default=0.6,
+                        help='Threshold for person instance segmentation model')
+    parser.add_argument('--output_video', type=str, default='', required=False,
+                        help='Optional. Path to output video')
+    parser.add_argument("--no_show", help="Optional. Don't show output", action='store_true')
+
+    parser.add_argument('-d', '--device', type=str, default='CPU')
+    parser.add_argument('-l', '--cpu_extension', type=str, default=None,
+                        help='MKLDNN (CPU)-targeted custom layers.Absolute \
+                              path to a shared library with the kernels impl.')
+    args = parser.parse_args()
+
+    if not path_exist(args.i, type='file') and not str.isdigit(args.i):
+        raise ValueError('File not found or wrong camera id: {}'.format(args.i))
+    else:
+        capture = VideoCapture(args.i)
+
+    if (args.m_instance_segmentation and args.m_semantic_segmentation) or \
+       (not args.m_instance_segmentation and not args.m_semantic_segmentation):
+        raise ValueError('Set up exectly one of segmentation models: '\
+                         '--m_instance_segmentation or --m_semantic_segmentation')
+
+    frame_size, fps = capture.get_source_parameters()
+    out_frame_size = (int(frame_size[0]), int(frame_size[1] * 2))
+
+    mouse = MouseClick()
+    cv2.namedWindow(WINNAME)
+    cv2.setMouseCallback(WINNAME, mouse.get_points)
+
+    if args.output_video:
+        fourcc = cv2.VideoWriter_fourcc(*'XVID')
+        output_video = cv2.VideoWriter(args.output_video, fourcc, fps, out_frame_size)
+    else:
+        output_video = None
+
+    log.info("Creating Inference Engine")
```

Comment:
"Initializing" or something, not "creating".

Reply:
Fixed.